src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -276,6 +276,9 @@ protected static T Cast<T>(object value)
             return currentDestination;
         }
 
+        private static CodeGenerationOptions ShouldGenerateMethodBody(CodeGenerationOptions options, IPropertySymbol property) =>
+            options.GenerateMethodBodies ? options.WithGenerateMethodBodies(property.IsIndexer || property.IsWriteOnly || property.IsReadOnly || property.Parameters.Length > 0) : options;
```

Comment:
Ideally, I think we'd want to have an option that allows the user to control whether or not they want the IDE to generate autoproperties when possible, and a refactoring to make "convert to autprop" to expand an autoprop into a full property.  Tagging @DustinCampbell and @CyrusNajmabadi explicitly to get their thoughts.